packages/flutter/lib/src/material/stepper.dart

```diff
@@ -0,0 +1,577 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+import 'package:meta/meta.dart';
+
+import 'button.dart';
+import 'colors.dart';
+import 'debug.dart';
+import 'flat_button.dart';
+import 'ink_well.dart';
+import 'material.dart';
+import 'theme.dart';
+
+/// The state of a step which is used to control the style of the circle and
+/// text.
+///
+/// /// See also:
+///
+///  * [Step]
+enum StepState {
+  /// A step that is disabled and does to react to taps.
+  disabled,
+  /// A step that is currently inactive, e.g. one that the user did not yet see.
+  inactive,
+  /// A step that is currently active, e.g. one that the user has seen and
+  /// edited.
+  active,
+  /// A step that is currently having an error. e.g. the use has submitted wrong
+  /// input.
+  error
+}
+
+const TextStyle _kStepStyle = const TextStyle(
+  fontSize: 12.0,
+  color: Colors.white
+);
+const TextStyle _kActiveTitleStyle = const TextStyle(
+  fontSize: 14.0,
+  fontWeight: FontWeight.w500,
+  color: Colors.black87
+);
+const TextStyle _kActiveSubtitleStyle = const TextStyle(
+  fontSize: 12.0,
+  color: Colors.black54
+);
+const TextStyle _kDisabledTitleStyle = const TextStyle(
+  fontSize: 14.0,
+  fontWeight: FontWeight.w500,
+  color: Colors.black38
+);
+const TextStyle _kDisabledSubtitleStyle = const TextStyle(
+  fontSize: 12.0,
+  color: Colors.black38
+);
+final TextStyle _kErrorTitleStyle = new TextStyle(
+  fontSize: 14.0,
+  fontWeight: FontWeight.w500,
+  color: Colors.red[500]
+);
+final TextStyle _kErrorSubtitleStyle = new TextStyle(
+  fontSize: 12.0,
+  color: Colors.red[500]
+);
+
+/// A Material step used in [Stepper]. The step can have a title and subtitle,
+/// an icon within its circle, some content and a state that governs its stying.
+///
+/// See also:
+///
+///  * [Stepper]
+///  * <https://material.google.com/components/steppers.html>
+class Step {
+  /// Creates a step.
```

Comment:
"Creates a step for a [Stepper]" maybe. Obviously a constructor called "step" creates a step. :-)